### ISSUE TITLE
[Screen Reader – Make Code - Report Abuse] Error message does not appear after providing invalid URL at 'What is the URL of the offensive project' edit field.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3637]
**User Experience**:
Providing information about how to correct input errors help users who have learning disabilities to fill in a fields successfully. Users who are blind or have impaired vision understand more easily the nature of the input error and how to correct it.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Report Abuse" option and select it.
7. Navigate through all the controls appearing under "Report Abuse" dialog.
8. Navigate to 'What is the URL of the offensive project' edit field and provide invalid input.
9. Verify the issue.

**Actual Result:**
Error message does not appear after providing invalid URL at 'What is the URL of the offensive project' edit field under 'Report Abuse' pop-up window.

**Expected Result:**
Error message should be appear as 'Invalid URL' after providing invalid URL at 'What is the URL of the offensive project' edit field under 'Report Abuse' pop-up window.

**Note:**
Same issue appear under Chrome & Firefox.

**WCAG Reference:**
[WCAG 3.3.3 - Error Suggestion](https://www.w3.org/WAI/WCAG21/Understanding/error-suggestion.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, NVDA, JAWS

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG3.3.3, #Win10-Edge(Chromium)

![WCAG3 3 3_Report Abuse_Error Suggestion](https://user-images.githubusercontent.com/72871089/98648172-ed96a100-235b-11eb-8c7f-62db7070935f.jpg)

**Comment by jwunderl** [comment]
entered url is checked on submission and user is notified at that point.
